config/locales/en.yml

```diff
@@ -616,12 +619,16 @@ en:
         is_optional_options:
           'false': Mandatory
           'true': Optional
+        is_repeatable_options:
+          'false': No — this question can only be answered once
```

Comment:
I wondered whether "this question can only be answered once" should be in hint text, matching the hint for Optional?

Reply:
@jamie-o-wilkinson this content is what was agreed with our content designer, see https://app.mural.co/t/gaap0347/m/gaap0347/1687946839058/9e1b89cd76af287c05c82df1cdb307a830324315?wid=0-1723651460259

Reply:
This is a good spot.

It's hard to link to, but this is one of the options Charlotte considered for the content of this page. You can see all the options in [the middle, bottom of this mural page](https://app.mural.co/t/gaap0347/m/gaap0347/1687946839058/9e1b89cd76af287c05c82df1cdb307a830324315?wid=237-1717167377834).

Basically, hint text for the option was considered and rejected. It's possible that could change in the future though as this will be tested with users soon.